agenta-backend/pyproject.toml

```diff
@@ -43,6 +43,9 @@ numpy = "1.26.3"
 autoevals = "^0.0.83"
 supertokens-python = "^0.24.2"
 
+protobuf = "^4.25.4"
```

Comment:
protobuf for this ?

Reply:
Copy pasted your pyproject.toml from observability so that we don't have any conflict with pyproject